src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -1214,6 +1216,22 @@ def getMaxPacketSize(self):
         else:
             return 32
 
+    def setMaxPacketSize(self, packetSize):
+        """
+        Sets the maximum packet size of sensor.
+
+        packetSize (int): 32, 64, 128 and 256 are supported.
+        """
+
+        try:
+            packetSizes = {32: 0, 64: 1, 128: 2, 256: 3}
+            packetMaxSizeType = packetSizes[packetSize]
+
+        except KeyError:
+            raise ValueError("Invalid packet size")
+
+        self.setSystemParameter(6, packetMaxSizeType)
```

Comment:
Maybe we should get rid of these magic numbers and use constants instead like `SYSTEM_PARAMETER_PACKAGE_SIZE = 5` - what do you think?

Reply:
Great idea! I often thought about it.

Reply:
But for consistence we should use `FINGERPRINT_SETSYSTEMPARAMETER` as prefix as it is related to this instruction code. I propose: `FINGERPRINT_SETSYSTEMPARAMETER_PACKAGE_SIZE = 5`

Reply:
:+1: